blocks/library/latest-posts/index.php

```diff
@@ -91,8 +91,7 @@ function register_block_core_latest_posts() {
 				'default' => 3,
 			),
 			'align'           => array(
-				'type'    => 'string',
-				'default' => 'center',
+				'type' => 'string',
```

Comment:
Oof... did we decide anything about how we're going to handle server attributes for supports properties, particularly if we're planning for server-registered attributes to become the canonical set?

Reply:
I think I will revert all changes for the latest posts block, as this is falls under special case group :) We can tackle it later.

Reply:
So this one is tricky. It is exposed with the global variable and consumed when the block is registered, but it is not that easy to ensure it behaves the same in the JS and PHP context. We might want to revisit the idea of using `has` check as I proposed earlier: https://github.com/WordPress/gutenberg/pull/5099#discussion_r168764070.

Reply:
>I think I will revert all changes for the latest posts block, as this is falls under special case group :) We can tackle it later.

I think we ought to at least have a strategy, because I'm starting to worry that these supports which add attributes may be non-viable altogether. Or do you think it's just a matter of duplicating the logic in the server?

Reply:
The issue here is that you need to define this attribute on the server to make it exposed to the `save` function when server-side rendering your block. This is a general issue with such blocks, there is lots of duplication. I think we should tackle all similar issues as a whole, not necessarily as part of this refactoring.

One way of dealing with it would be to mark a component as handled server-side. However it still doesn't solve the issue of code duplication, because we end up in a situation where `edit` is handled client-side, but `save` mostly server-side. I don't think we can completely switch over to the solution where this is cleanly separated. We can only mitigate the unexpected side-effects where we set something on the client but it isn't passed back to the server.

Reply:
>I think we should tackle all similar issues as a whole, not necessarily as part of this refactoring.

I'm personally considering it a blocker because I'm not convinced yet there's a solution at all.

Reply:
@youknowriad, @mtias  or @mcsf, any ideas how to unblock it? :) The only thing I can come up is to avoid adding `align` to `attributes` if it was already provided server-side, but it doesn't solve the root of the problem. We need to make some decisions how do we handle server-side rendered blocks. The main question is how much duplication are we happy to introduce between PHP and JS?